.github/workflows/ci.yml

```diff
@@ -36,5 +36,5 @@ jobs:
 
       - name: Run Forge tests
         run: |
-          forge test -v --fork-url=${{secrets.NODE}} --fork-block-number=15035420
+          forge test -vvv --fork-url=https://rpc.ankr.com/eth --fork-block-number=15035420
```

Comment:
Should we change it to the polygon node?

wdyt?

Reply:
We should, but it interferes with changes in your PR, so I wanted to create less merge conflicts